syntax = "proto3";

//客户端
message RequestMsg
{
    //1 密钥协商 //2 密钥校验 //3 密钥注销
    int32 cmdType = 1;//报文类型
    bytes clientID = 2;//客户端编号
    bytes serverID = 3;//服务端编号
    string sign = 4;
    string data = 5;
}

//服务端
message RespondMsg
{
    int32 status = 1;//返回值
    int32 seckeyID = 2;
    bytes clientID = 3;//客户端编号
    bytes serverID = 4;//服务端编号
    string data = 5;//服务器段随机数
}